### PR TITLE
Added text message reminders

### PR DESCRIPTION
**Pull request by donovanlopez**
I added text message reminders. I tested it and it should work but there's a problem with Heroku where when the Heroku stays idle, it stops all processes and then stops the cron jobs.

### CODE REVIEW

File: text-messaging/send_sms.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+var cronJobs = {};
+
+module.exports = (  ) => {
+
+  console.log("In send_sms.js ");
+
+  //initial start of jobs 
+  main();
+
+  //at midnight, recheck the jobs
```
Comment (trevhud) [inline text-messaging/send_sms.js:16]: @donovanope Did you end up getting this part working?

File: text-messaging/send_sms.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        if (Object.keys(cronJobs).includes(client.name))
+        {
+          //for debugging
+          // console.log("new schedule for " + client.name + " at " + client.schedule_seeding + " and " + client.schedule_daily_checkups);
+          
+          cronJobs[client.name]['seeding'].setTime(new CronTime(client.schedule_seeding, "America/Los_Angeles"));
+          cronJobs[client.name]['daily_check'].setTime(new CronTime(client.schedule_daily_checkups, "America/Los_Angeles"));
+        
+          cronJobs[client.name]['seeding'].start();
+          cronJobs[client.name]['daily_check'].start();
```
Comment (trevhud) [inline text-messaging/send_sms.js]: Could probably chain these functions

**Review by trevhud — COMMENTED** [review]
Look great @donovanope . @edo92 Can you review and approve if good to go?

File: text-messaging/send_sms.js (hunk 1 above)
Comment (donovanlopez) [reply text-messaging/send_sms.js:16]: It works but when it runs on Heroku for like 30 minutes with nothing happening all processes get shut down. I'm looking up what to do to fix it. But other than that, it should work fine.

File: text-messaging/send_sms.js (hunk 2 above)
Comment (donovanlopez) [reply text-messaging/send_sms.js]: I tried but it threw an error.

**Pull request closed without merging**